pkgs/development/python-modules/paste/default.nix

```diff
@@ -1,34 +1,42 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 , six
-, pytest-runner
-, pytest
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "paste";
   version = "3.5.0";
 
-  src = fetchPypi {
-    pname = "Paste";
-    inherit version;
-    sha256 = "17f3zppjjprs2jnklvzkz23mh9jdn6b1f445mvrjdm4ivi15q28v";
+  src = fetchFromGitHub {
+    owner = "cdent";
+    repo = "paste";
+    rev = version;
+    sha256 = "sha256-yaOxbfQ8rdViepxhdF0UzlelC/ozdsP1lOdU5w4OPEQ=";
   };
 
-  propagatedBuildInputs = [ six ];
+  postPatch = ''
+    patchShebangs tests/cgiapp_data/
+  '';
 
-  checkInputs = [ pytest-runner pytest ];
+  # propagatedBuildInputs = [ six ];
```

Comment:
What's up with that change? It has no explanation provided either in comments or in the commit description.

FYI this broke the build of a Python module I'm packaging (but haven't upstreamed), so I suspect it's problematic.